release-notes/3.1/releases.json

```diff
@@ -4,7 +4,7 @@
   "latest-release-date": "2021-12-14",
   "latest-runtime": "3.1.22",
   "latest-sdk": "3.1.416",
-  "support-phase": "lts",
+  "support-phase": "maintenance",
```

Comment:
I think this one still causes me pause. Isn't 3.1 technically still an LTS release from a support phase standpoint?  I know given 6.0 GA now it is the newest LTS...but feels like this is still under a technical LTS support-phase?

Reply:
3.1 is now on maintenance mode too from a servicing/support mode, meaning it only gets security bug fixes on its last year (see https://github.com/dotnet/core/blob/main/release-policies.md).

That's why I keep advocating that we need to separate release type (lts or current) from support phase (full support, maintenance, and end of life).

Reply:
I feel like the only change needed is to change the json format of these from just a single string for support-phase to an array of json strings instead so it can be updated for cases like for 6.0 now:

```json
// snip.
    "support-phase": [ "current", "lts" ],
// snip.
```